change/@fluentui-react-components-bc19f521-d33b-4fa7-bfa8-a30b797d6b83.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "to unstable",
+  "packageName": "@fluentui/react-components",
```

Comment:
Looks like there are double change files somehow for both react-components and react-search